### PR TITLE
completed html-forms exercise - Colin Macrae 

### PR DESCRIPTION
**Pull request by colin-macrae**

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  <input type="text" name="subscribe" id="email"/>
+  <div><label for="#codingCool">Why is coding cool?</label></div>
+  <select name="options" id="codingCool">
+    <option value="Select an option">Select an option</option>
+    <option value="Because it's interesting">Because it's interesting</option>
+    <option value="Because it's fun">Because it's fun</option>
+    <option value="Because it's powerful">Because it's powerful</option>
+  </select>
+  <div>
+    <p></p>
```
Comment (JenBowman1) [inline html-forms/index.html:18]: please remove empty elements if you're not using them!

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <label for="#favLang">Favorite Language (choose one)</label>
+    <p></p>
```
Comment (JenBowman1) [inline html-forms/index.html:20]: see above

**Review by JenBowman1 — CHANGES_REQUESTED** [review]
just a couple small changes!

File: html-forms/index.html (hunk 1 above)
Comment (colin-macrae) [reply html-forms/index.html:18]: Okay. I only used them to create space.  Could I delete them and change that wrapped label into a <p>?

Comment (colin-macrae) [reply html-forms/index.html:18]: into a p element

**Comment by colin-macrae** [comment]
Updated.

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**